<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageWindow</class>
 <widget class="QDialog" name="ImageWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>550</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>동작 편집</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog{
	background-color: #EFEFEF;
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>340</width>
     <height>530</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: white;
border:1px solid black;
border-radius: 5px;
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QPushButton" name="button_save">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>저장</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button_cancel">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>490</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
      <hintingpreference>PreferVerticalHinting</hintingpreference>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    border-radius: 5px;
	color:black;
    border: 1px solid #8f8f91;
    background-color: #EFEFEF;
    padding: 2px 4px;
}</string>
    </property>
    <property name="text">
     <string>취소</string>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::InsertText"/>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_tip">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>300</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
color:black;</string>
    </property>
    <property name="text">
     <string>화면 내에서 이미지가 인식될 부분을 선택하세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_preview">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>300</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
	background-color: #EAEFEF;
	border: 1px solid black;
}</string>
    </property>
    <property name="text">
     <string>이곳을 클릭하여
이미지를 선택하세요</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="button_reset">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>30</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>88</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background: white;
color:black;
border: 1px solid black;
border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
    </property>
   </widget>
   <widget class="QLabel" name="label_surface2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>310</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>128</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: darkgray;
color: white;

border-top: 1px solid black;
border-bottom: 1px dashed white;
border-left: 1px dashed white;
border-right: 1px solid black;

border-top-left-radius :0px;
border-top-right-radius : 5px;
border-bottom-left-radius : 0px;
border-bottom-right-radius : 0px;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>2</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_surface1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>310</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>128</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: deepskyblue;
color: white;

border-top: 1px solid black;
border-bottom: 1px dashed white;
border-left: 1px solid black;
border-right: 1px dashed white;

border-top-left-radius :5px;
border-top-right-radius : 0px;
border-bottom-left-radius : 0px;
border-bottom-right-radius : 0px;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_surface3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>128</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: darkgray;
color: white;

border-top: 1px dashed white;
border-bottom: 1px solid black;
border-left: 1px solid black;
border-right: 1px dashed white;

border-top-left-radius :0px;
border-top-right-radius : 0px;
border-bottom-left-radius : 5px;
border-bottom-right-radius : 0px;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_surface4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>380</y>
      <width>90</width>
      <height>70</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>128</horstretch>
      <verstretch>60</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: darkgray;
color: white;

border-top: 1px dashed white;
border-bottom: 1px solid black;
border-left: 1px dashed white;
border-right: 1px solid black;

border-top-left-radius :0px;
border-top-right-radius : 0px;
border-bottom-left-radius : 0px;
border-bottom-right-radius : 5px;</string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>4</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>300</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
border-radius: 5px; 
padding-left:10px</string>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32767</number>
    </property>
    <property name="placeholderText">
     <string>동작의 이름을 입력하세요</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
